### PR TITLE
Added synchonous machine and user setting

### PR DESCRIPTION
**Pull request by AlexSP0**
partially edited package settings

### CODE REVIEW

File: en-US/basealtpackages.adml [hunk 1]
```diff
@@ -29,14 +29,31 @@ Disabled - Don't update OS automatically
 
 Enabled - Update OS automatically
       </string>
+
+      <string id="sync_packages_work_machine">Synchronous work with packages</string>
+      <string id="sync_packages_work_machine_help">Enable this setting for synchronous work with packages</string>
+
+      <string id="sync_packages_work_user">Synchronous work with packages</string>
+      <string id="sync_packages_work_user_help">Enable this setting for synchronous work with packages</string>
```
Comment (mastersin) [inline en-US/basealtpackages.adml:37]: Аналогично "Synchronous work with packages" - "Enable this setting for synchronous work with packages".

**Review by mastersin — CHANGES_REQUESTED** [review]

File: BaseALTPackages.admx [hunk 2]
```diff
@@ -14,21 +14,39 @@
...
       <elements>
-        <list id="InstallPackagesList" key="Software\BaseALT\Policies\Packages\Install" valuePrefix="" />
+        <list id="InstallPackagesList" key="Software\BaseALT\Policies\Packages\Install" additive="true"/>
       </elements>
     </policy>
-    <policy class="Machine" displayName="$(string.packages_remove)" explainText="$(string.packages_remove_help)" key="Software\BaseALT\Policies\Packages" name="RemovePackages" presentation="$(presentation.RemovePackages)">
+
+    <policy class="User" displayName="$(string.packages_install)" explainText="$(string.packages_install_help)" key="Software\BaseALT\Policies\Packages" name="InstallPackagesUser" presentation="$(presentation.InstallPackagesUser)">
+      <parentCategory ref="ALT_Packages"/>
+      <supportedOn ref="system:SUPPORTED_AltP8"/>
```
[the remark this answers is not in the document]
Comment (AlexSP0) [reply BaseALTPackages.admx]: fixed

File: BaseALTPackages.admx [hunk 3]
```diff
@@ -14,21 +14,39 @@
   </categories>
 
   <policies>
-    <policy class="Machine" displayName="$(string.packages_install)" explainText="$(string.packages_install_help)" key="Software\BaseALT\Policies\Packages" name="InstallPackages" presentation="$(presentation.InstallPackages)">
+    <policy class="Machine" displayName="$(string.packages_install)" explainText="$(string.packages_install_help)" key="Software\BaseALT\Policies\Packages" name="InstallPackagesMachine" presentation="$(presentation.InstallPackagesMachine)">
```
[the remark this answers is not in the document]
Comment (AlexSP0) [reply BaseALTPackages.admx:17]: fixed

**Review by mastersin — APPROVED** [review]
LGFM

**Pull request merged**